package.json

```diff
@@ -12,37 +12,31 @@
     "prenetlify": "npm run build",
     "netlify": "node scripts/netlify.js",
     "prebuild": "npm run clean",
-    "build": "run-s build:webworker build:js",
+    "build": "npm-run-all -p build:*",
     "build:js": "rollup -c scripts/rollup.config.js",
-    "build:webworker": "rollup -c scripts/webworker.rollup.config.js",
-    "clean": "run-p clean:dist clean:test",
-    "clean:dist": "rimraf dist dist-test",
-    "clean:test": "node scripts/clean-tests.js",
-    "postclean": "mkdirp dist",
-    "docs": "run-p docs:*",
-    "docs:api": "jsdoc src -r -d docs/api",
-    "docs:toc": "doctoc README.md",
-    "docs:nomnoml:build": "node -e \"var b=require('./scripts/nomnoml.js'); b.build();\"",
+    "clean": "shx rm -rf ./dist ./test/dist",
+    "postclean": "shx mkdir -p ./dist ./test/dist",
+    "build:test-manifests": "node ./scripts/create-test-manifests.js",
```

Comment:
I don't think these two need to be run on `npm run build`, just on test should be enough

Reply:
I would also prefer not to build the test dist file on just regular build but that sounds like it should happen in a separate PR

Reply:
Well since we build test assets on build we have to do them on build for now.